docs/tools/cli/byoc.md

```diff
@@ -0,0 +1,127 @@
+| `--byoc-id`         | Yes      | Identifier of a custom cloud for which to retrieve permissions details                                                |
+
+### `avn byoc cloud permissions remove`
+
+Revokes permissions to a custom cloud.
+
+| Parameter           | Required | Information                                                                                                           |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where a custom cloud is located                                                         |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud for which to revoke permissions                                                          |
+| `--account`         | No       | Identifier of your account                                                                                            |
+| `--project`         | No       | Name of the project for which to revoke permissions                                                                   |
+
+### `avn byoc cloud permissions set`
+
+Modifies permissions to a custom cloud.
+
+| Parameter           | Required | Information                                                                                                           |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where a custom cloud is located                                                         |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud for which to set permissions                                                             |
+| `--account`         | No       | Identifier of your account                                                                                            |
+| `--project`         | No       | Name of the project for which to set permissions                                                                      |
+
+## Manage an infrastructure template
+
+Manage a custom cloud Terraform infrastructure template.
+
+### `avn byoc templete terraform get-template`
+
+Downloads a custom cloud Terraform template.
+
+| Parameter           | Required | Information                                                             |
+| ------------------- | -------- | ----------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where a custom cloud is located           |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud for which to download a Terraform template |
+
+### `avn byoc templete terraform get-vars`
+
+Downloads a custom cloud variables file.
```

Comment:
We could call it a "Terraform variables file" just to be more precise and match TF's own documentation.